dev/set_matrix.py

```diff
@@ -204,7 +208,10 @@ def get_changed_flavors(changed_files, flavors):
     """
     changed_flavors = []
     for f in changed_files:
-        match = re.search(r"^(mlflow|tests)/(.+?)(\.py|/)", f)
+        pattern = r"^(mlflow|tests)/(.+?)(_autolog(ging)?)?(\.py|/)"
+        #                           ~~~~~
+        #                           # This group captures a flavor name
+        match = re.search(pattern, f)
```

Comment:
Maybe we should fix folder names instead.

Reply:
I think it's better to cast a wider net and capture `_autolog` in addition to `_autologging`. LGTM :)